### PR TITLE
[stacktrace-js] add getSync definition

### PR DESCRIPTION
**Pull request by pilagod**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/stacktrace-js/index.d.ts [hunk 1]
```diff
@@ -47,6 +48,13 @@ declare namespace StackTrace {
   export function get(options?: StackTraceOptions): Promise<StackFrame[]>;
 
   /**
+   * Get a backtrace from invocation point synchronously.
+   * @param options Options Object
+   * @return Array[StackFrame]
+   */
+  export function getSync(options?: { filter?: (stackFrame: StackFrame) => boolean }): StackFrame[]
```
Comment (yuit) [inline types/stacktrace-js/index.d.ts]: I am a bit surprised this is not the `StackTraceOptions` But I also see some inconsistent in the original library 

- https://github.com/stacktracejs/stacktrace.js/blob/1e3acf53c358e01a05ea9f68a445f7f76e1f93f4/stacktrace-js.d.ts#L83 

- https://github.com/stacktracejs/stacktrace.js/blob/1e3acf53c358e01a05ea9f68a445f7f76e1f93f4/README.md#stacktracegetsyncoptional-options--arraystackframe

**Comment by yuit** [comment]
also since there is already .d.ts with the pkg (https://github.com/stacktracejs/stacktrace.js/blob/1e3acf53c358e01a05ea9f68a445f7f76e1f93f4/stacktrace-js.d.ts) should this .d.ts be removed?

File: types/stacktrace-js/index.d.ts (hunk 1 above)
Comment (pilagod) [inline types/stacktrace-js/index.d.ts]: I didn't realise possibility of inconsistency in the pkg, I only see those specs defined in README, but missed the .d.ts file 😂.

**Comment by pilagod** [comment]
Although there is already a .d.ts in pkg, it's a little trivial to reference to it in every files, or change tsconfig for it. 
But it indeed seems to be a duplicate work here, do u have any idea to cope with this situation ?

**Comment by pilagod** [comment]
Nevertheless, I make a revision first 🙂️

**Comment by pilagod** [comment]
Seeing the [header comments](https://github.com/stacktracejs/stacktrace.js/blob/1e3acf53c358e01a05ea9f68a445f7f76e1f93f4/stacktrace-js.d.ts#L4), it seems to reference to definitions in this repo

**Comment by yuit** [comment]
@pilagod  I see.... I will merge the update in and if in the future they decide to bundle with .d.ts we can come in and remove...Thanks for the PR!

**Comment by pilagod** [comment]
@yuit Really thanks for your help 😃 Let's see how things go next !

**Pull request merged**